src/legacy/ui/public/courier/fetch/components/_shard_failure_modal.scss

```diff
@@ -0,0 +1,50 @@
+// set width and height to fixed values to prevent resizing when you switch tabs
+.shardFailureModal {
+     height: 75vh;
+     width: 768px;
+}
+
+// displaying a EuiCodeBlock inside a tab inside a modal, you've to set the width 
+// to prevent overlapping the modal when the content contains very long lines
+.shardFailureModal__codeblock {
+    width: 702px
+}
+
+// making EuiDescriptionList a bit more compact, there's not so much space inside a modal
+.euiDescriptionList.euiDescriptionList--column {
+    .shardFailureModal__descTitle {
+        width: 20%;
+        line-height: 1.1em;
+        margin-top: 0.8em;
```

Comment:
We try to stay away from using `em` values and stick to using SASS vars. This number equates to about `11px` so you can instead use `$euiSizeM` which equals `12px` or go smaller with `$euiSizeS` which is `8px`. We try to stick to a nice 4x4 grid pattern.

Reply:
thx, will change